builder/partials/body.hbs

```diff
@@ -29,14 +29,19 @@
 
 {{#unless hideHeader}}
 <footer>
-    <style>
-        .add-snippet-button {
-            position: fixed;
-            bottom: 20px;
-            left: 84%;
-            z-index: 10;
-        }
-    </style>
+    <div class="card">
+        {{#with statistics }}
+            <p>
+                Statistics: {{ snippets }} snippets, {{ authors }} authors, {{ tags }} tags.
+            </p>
+            <p>
+                Generated {{ generationDate }}, based on commit
+                <a href="https://github.com/nycJSorg/30-seconds-of-angular/tree/{{ revision }}" target="_blank">{{ revisionAbbr }}</a>
```

Comment:
Can we have it in site metadata? We're going to move the repo soon

Reply:
I don't think this information belongs to metalsmith meta. It's much easier to change this in template.

Reply:
`https://github.com/nycJSorg/30-seconds-of-angular ` this part certainly should not be it 10 different templates